src/main/scala/org/allenai/pipeline/Producer.scala

```diff
@@ -106,8 +123,11 @@ trait CachingDisabled extends CachingEnabled {
   override def cachingEnabled: Boolean = false
 }
 
-class PersistedProducer[T, -A <: Artifact](step: Producer[T], io: SerializeToArtifact[T, A] with DeserializeFromArtifact[T, A],
-  _artifact: A) extends Producer[T] {
+class PersistedProducer[T, -A <: Artifact](
```

Comment:
I'm not sure we do.  The time taken should be short--why not just compute the time taken?  It would be nice to mention when data is loaded from an intermediate file, and where that file is.  Let's talk a bit this morning.